recipes/compas_convert/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "compas_convert" %}
+{% set version = "0.1.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8b0c7544930b134294a1d0f8eaaffe62a930decc1c26b3ce2816f010ccecc8d3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vvv
+
+requirements:
+  host:
+    - build
```

Comment:
`script` above is using `pip` rather than `build` so I don't think you should need this?
```suggestion
```

Reply:
You're right!  I've gotten my build processes confused!